src/adres-crab/adres-crab.ts

```diff
@@ -81,7 +78,7 @@ export class AdresCrab {
   }
 
   public gemeenteChanged() {
-    if (!this.isVlaamseProvincie(this.data.gemeente.provincie)) {
+    if (this.data.gemeente && this.data.gemeente.provincie && !this.isVlaamseProvincie(this.data.gemeente.provincie)) {
```

Comment:
hmm, als de provincie Vlaams is meoten we toch juist wel de suggest autocomplete hebben? of lees ik de code verkeerd?

Reply:
De `autocompleteType.Suggest` betekent dat als er iets gevonden is gaan we die tonen maar gebruiker is niet verplicht om enkel uit de lijst te kiezen. Bij `autocompleteType.Auto` niks buiten de lijst mag ingevuld worden.